zkevm-circuits/src/keccak_circuit/keccak_packed_multi.rs

```diff
@@ -574,10 +574,9 @@ pub(crate) fn keccak<F: Field>(
                 transform::value(&mut cell_manager, &mut region, packed, false, |v| *v, true);
             cell_manager.start_region();
             let mut is_paddings = Vec::new();
-            let mut data_rlcs = Vec::new();
+            let mut data_rlcs = vec![Value::known(F::zero()); get_num_rows_per_round()];
```

Comment:
```suggestion
            let mut data_rlcs = vec![Value::known(F::zero()); NUM_BYTES_PER_WORD + 1];
```
I think this is the exact length we want for `data_rlcs`.

Reply:
This array holds a value for each row of the round, including rows those not involved in the RLC computation, to be used here:

```rust
            for row_idx in 0..get_num_rows_per_round() {
                rows.push(KeccakRow {
                    q_enable: row_idx == 0,
                    q_round: row_idx == 0 && round < NUM_ROUNDS,
                    q_absorb: row_idx == 0 && round == NUM_ROUNDS,
                    q_round_last: row_idx == 0 && round == NUM_ROUNDS,
                    q_padding: row_idx == 0 && round < NUM_WORDS_TO_ABSORB,
                    q_padding_last: row_idx == 0 && round == NUM_WORDS_TO_ABSORB - 1,
                    round_cst,
                    is_final: is_final_block && round == NUM_ROUNDS && row_idx == 0,
                    length: round_lengths[round],
                    data_rlc: round_data_rlcs[round][row_idx],
                    hash_rlc,
                    cell_values: regions[round].rows[row_idx].clone(),
                });
            }
```

Reply:
Ah I missed that, sorry; you're right!